controller/server/view/common/page.ml

```diff
@@ -55,28 +64,43 @@ let html ?current_page content =
         ]
     )
     (body
-      ~a:[ a_class [ "d-Container" ] ]
-      [ header ~a:[ a_class [ "d-Header" ] ] []
-      ; nav
-          ~a:[ a_class [ "d-Menu" ] ]
-          ([ Info; Network; Localization; Shutdown ]
-              |> List.map (menu_item current_page))
-      ; main
-          ~a:[ a_class [ "d-Content" ] ]
-          [ content ]
-      ; footer
-          ~a:[ a_class [ "d-Footer" ] ]
-          [ a
-              ~a:[ a_href "/changelog"
-              ; a_class [ "d-Footer__Link" ]
-              ]
-              [ txt "changelog" ]
-          ; a
-              ~a:[ a_href "/status"
-              ; a_class [ "d-Footer__Link" ]
-              ]
-              [ txt "system status" ]
-          ]
+      ~a:[ a_class [ "d-Layout" ] ]
+      (( aside
+          ~a:[ a_class [ "d-Layout__Aside" ] ]
+          [ nav
+              ([ Info; Network; Localization; SystemStatus; Changelog ]
+                |> List.map (menu_item current_page))
+          ; div
+              ~a: [ a_class [ "d-Layout__Shutdown" ] ]
+              [ menu_item current_page Shutdown ]
+          ]) 
+      :: header 
+      @ [ main
+              ~a:[ a_class [ "d-Layout__Main" ] ]
+              [ content ]
       ; script ~a:[ a_src "/static/client.js" ] (txt "")
-      ]
+      ])
     )
+
+let header_title ?back_url ?icon ?right_action content =
```

Comment:
It would make better sense for me if this was called simply `header` as the `title` seems to be a child element of this.